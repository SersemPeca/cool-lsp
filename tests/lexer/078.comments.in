"hello"
(* multi-line comments
    (* need to be terminated even after nesting
