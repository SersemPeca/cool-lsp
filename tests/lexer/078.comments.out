#1 STR_CONST "hello"
#3 ERROR: Unmatched (*
